draft-enghardt-panrg-path-properties.md

```diff
@@ -212,14 +196,14 @@ Protocol Features available:
 : Whether a specific protocol feature is available over a path or subpath, e.g., Explicit Congestion Notification (ECN), or TCP Fast Open.
 
 
-# Dynamic Properties
+## Dynamic Path Properties
 
 Dynamic path properties relate to the transmission of an individual packet or of a flow over a subpath.
```

Comment:
As said in the first comment, maybe we should change this since this also includes some non-dynamic path properties.